contracts/lib/HashLib.sol

```diff
@@ -47,21 +48,23 @@ bytes32 constant MULTICHAIN_SESSION_TYPEHASH = keccak256(
     )
 );
 
-/// @dev `keccak256("EIP712Domain(string name,string version,uint256 chainId,address verifyingContract)")`.
-bytes32 constant _DOMAIN_TYPEHASH = 0x8b73c3c69bb8fe3d512ecc4cf759cc79239f7b179b0ffacaa9a75d522b39400f;
+//0x8b73c3c69bb8fe3d512ecc4cf759cc79239f7b179b0ffacaa9a75d522b39400f
+bytes32 constant _DOMAIN_TYPEHASH = keccak256("EIP712Domain(string name,string version,uint256 chainId,address verifyingContract)");
```

Comment:
should we address this?

https://cantina.xyz/code/cdf37612-211c-4472-9e35-f33b9a4802a4/findings/18?status=new%2Cdisputed%2Crejected%2Cduplicate%2Cconfirmed%2Cacknowledged%2Cfixed&with_events=false&limit=20

Reply:
yes, will address this in this PR